x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/integration/RoleMappingFileSettingsIT.java

```diff
@@ -307,48 +315,78 @@ public void testRoleMappingsApplied() throws Exception {
             clusterStateResponse.getState().metadata().persistentSettings().get(INDICES_RECOVERY_MAX_BYTES_PER_SEC_SETTING.getKey())
         );
 
-        // native role mappings are not affected by the removal of the cluster-state based ones
+        // cluster-state role mapping was removed and is not returned in the API anymore
```

Comment:
is this testing that the cluster state role mappings are added then deleted as expected ? if so, can you rename this to `testClusterStateRoleMappingsAddedThenDeleted` (or the like)